src/static/courses.ts

```diff
@@ -1,7 +1,7 @@
 export const courseNames = {
   web3: 'web3',
   solana: 'solana',
-  near: 'near'
+  near: 'NEAR'
```

Comment:
I think I left these lowercase for a reason. The `solana` right above this is not capitalized. It doesn't seem to affect anything so it's probably fine - but the `courseTitles` right below this are for the branding/display

```suggestion
  near: 'NEAR'
```
```suggestion
  near: 'near'
```

Reply:
But the `courseName` is used for the Superblock page